notebooks/257-llava-multimodal-chatbot/257-llava-multimodal-chatbot.ipynb

```diff
@@ -0,0 +1,1222 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #154.                @wraps(orig_forward)
</pre><p>Maybe define wrapper outside of the function body.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1353/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I do not think that it is possible, as wrapped function unknown outside of function, I can only move whole patching in separated function